contentcuration/contentcuration/static/js/edit_channel/clipboard/hbtemplates/clipboard_step_1.handlebars

```diff
@@ -0,0 +1,23 @@
+<div id="clipboard" class="pull-right">
+	<div class="clipboard_header">
+		<h3 class="text-center">
+			<img src="/static/img/add_folder.png" alt="add" class="pull-left"/>
+			Adding Content to Topic
+			<a class="back pull-right btn-link toggle_clipboard">X</a>
+		</h3>
+		<table>
+			<tr>
+				<td class="text-center"><h4>1</h4><h6>Source</h6></td>
```

Comment:
You should use an HTML ordered list here, so that the item labels are not hardcoded.